Framework/DataHandling/src/SaveNXcanSAS.cpp

```diff
@@ -722,26 +759,36 @@ void SaveNXcanSAS::init() {
   declareProperty(std::make_unique<Mantid::API::FileProperty>("Filename", "", API::FileProperty::Save, ".h5"),
                   "The name of the .h5 file to save");
 
-  std::vector<std::string> radiation_source{"Spallation Neutron Source",
-                                            "Pulsed Reactor Neutron Source",
-                                            "Reactor Neutron Source",
-                                            "Synchrotron X-ray Source",
-                                            "Pulsed Muon Source",
-                                            "Rotating Anode X-ray",
-                                            "Fixed Tube X-ray",
-                                            "neutron",
-                                            "x-ray",
-                                            "muon",
-                                            "electron"};
+  std::vector<std::string> radiationSourceOptions{"Spallation Neutron Source",
+                                                  "Pulsed Reactor Neutron Source",
+                                                  "Reactor Neutron Source",
+                                                  "Synchrotron X-ray Source",
+                                                  "Pulsed Muon Source",
+                                                  "Rotating Anode X-ray",
+                                                  "Fixed Tube X-ray",
+                                                  "neutron",
+                                                  "x-ray",
+                                                  "muon",
+                                                  "electron"};
   declareProperty("RadiationSource", "Spallation Neutron Source",
-                  std::make_shared<Kernel::StringListValidator>(radiation_source), "The type of radiation used.");
+                  std::make_shared<Kernel::StringListValidator>(radiationSourceOptions), "The type of radiation used.");
+  std::vector<std::string> geometryOptions{"Cylinder", "FlatPlate", "Disc"};
```

Comment:
I'm still learning about C++ imports so this may not be a good suggestion, but now that we have constants for these geometry options defined in Sample.h would it be worth using them here?